test/run-end-to-end.sh

```diff
@@ -0,0 +1,18 @@
+set -e
+if ! docker volume inspect toolsdata >/dev/null; then 
+    docker volume create toolsdata && \
+    docker run --rm  -w / -v toolsdata:/switchboard-tool-registry alpine/git \
+        clone https://github.com/clarin-eric/switchboard-tool-registry/ --branch production --depth 1
```

Comment:
I am not sure using a full docker image to only to do a git checkout is the best solution here. Did you consider relying on the running platform's `git`?

Reply:
I will try that, I thought at first to have a solution based only on docker, but it may be overkill

Reply:
I do see the advantage of this approach, as it always "just works" provided there is internet connection. But on the other hand there are 2 disadvantages: there is indeed the fact that we pull a lot of extra unecessary data but also the usual concern that referring to images like: `alpine/git` is known to be potentially insecure.

Reply:
I missed this detail. So we are explicitly tying this to a specific branch, rather than the 'context' branch of the job?

Reply:
No, this is the tool registry branch. i.e. the version of the tools list that will be used for the tests.